gpsreceiver/gpsreceiver/antenna.py

```diff
@@ -95,13 +95,14 @@ def __init__(self, receiver: Receiver) -> None:
         # This attribute is used to store the leftover samples which are
         # prepended to the next chunk of samples and forwarded to the receiver.
         self._samples: Samples | None = None
+        self.gain = gain
```

Comment:
I'm guessing this variable is intended to be private, in which case could you please prefix it with an underscore to match the style used elsewhere in the code? A type annotation would be good too.

```suggestion
        self._gain: int = gain
```